core/modules/admin_bar/tests/admin_bar.test

```diff
@@ -490,4 +490,60 @@ class AdminBarCustomizedTestCase extends AdminBarWebTestCase {
       ':path' => $path,
     ))->fetchField();
   }
+  /**
+   * Test the "Back To Site" link functionality.
+   *
+   * @broken
+   * @todo Figure out how to test for HTML in the admin bar. The debug output for
+   *   the failed tests doesn't seem to give me that information.
+   */
+  /*
```

Comment:
Remove this.

Reply:
@klonos, the whole test needs to be removed (not just the commenting line). This test always fails because the replacement of the icon with "Back to site" is done by Javascript on the client side. So, at the moment, it's impossible to test this functionality (with the tests we use in Backdrop). 

I think @keiserjb is aware of this.

Reply:
Test removed.  It apparently did not work.